README.md

```diff
@@ -101,10 +112,28 @@ a template to create your ``.env`` file. For MicroMaster’s to work, it needs 3
     on port ``8000`` by default.
 - ``EDXORG_CLIENT_ID`` and ``EDXORG_CLIENT_SECRET``
 
-    These values can be found in Django admin OAuth client section discussed above.
+    These values can be found in the Django OAuth Toolkit Application you created above.
     **Client id:** and **Client secret:** values should be auto-generated for
-    your new client. Use those values for the corresponding ``EDXORG_``
-    variables in the ``.env`` file
+    that new Application. Use those values for the corresponding ``EDXORG_``
+    variables in the ``.env`` file.
+
+#### General edX devstack debugging notes
+
+- To update your devstack with important changes from edX, run `vagrant provision` in
+your edx_devstack directory. This will pull down the latest release and run migrations, among
+other things.
```

Comment:
Will running `vagrant provision` on a freshly installed devstack pull the current master, or something else?